chall02/cdorta-n.py

```diff
@@ -0,0 +1,40 @@
+#/usr/bin/python3
```

Comment:
It's not a good way to write a shebang, I invite you to read documentation about it, it'll be fun: there is some debate about it